src/scss/styles.scss

```diff
@@ -740,6 +742,11 @@ app-user-billing {
     }
 }
 
+.form-check-label {
+    margin: 0.3rem 0 0 0.3rem;
+    line-height: 16px;
+}
```

Comment:
We use bootstrap for these styles. Maybe you should propose these fixes upstream?

Reply:
For the icons and check boxes align, `line-height: 1.5;` is bootstrap property declared on body element that is making the mess. From my experience, those are the common Bootstrap inconsistencies. Please, check out the difference at last row (maybe not best example):

![2020-05-04-18-21-vault bitwarden com](https://user-images.githubusercontent.com/28996115/80992134-59da9a00-8e39-11ea-8abc-2875a144501d.png)

By the way, icons vertical alignment at `secure notes` mobile app section are also a mess. Really looks bad.  

When checking the form label `Remember email` styles on live site, it's seems that changes does not have the same effect as in my local app version. Please, take a look at my local version with changes:

![Screenshot_2020-05-04 Bitwarden Web Vault](https://user-images.githubusercontent.com/28996115/80992386-b63db980-8e39-11ea-8221-6493a7f71ae3.png)

Live site with same changes in dev tools:

![2020-05-04-19-02-vault bitwarden com](https://user-images.githubusercontent.com/28996115/80992533-f0a75680-8e39-11ea-8b24-1f478b2e97d8.png)

> Maybe you should propose these fixes upstream?

Sorry, but I can't figure it out what do you mean? I will do what ever it takes.